AuthorizationService/Authorization.Domain/DataTransferObject/RegisterAccountDto.cs

```diff
@@ -0,0 +1,25 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Authorization.Domain.DataTransferObject
+{
+    public class RegisterAccountDto
+    {
+        [Required]
+        public string Email { get; set; }
+        [Required]
+        [DataType(DataType.Password)]
+        public string Password { get; set; }
+        [Required]
+        [DataType(DataType.Password)]
+        [Compare("Password")]
+        public string ConfirmPassword { get; set; }
+        public string UserName { get { return Email; } }
```

Comment:
Consider mapping of Email to UserName inside mapping profile and delete UserName from this Dto